src/core/diags/factory_diags.cpp

```diff
@@ -577,6 +577,16 @@ Error Diags::ProcessRadio(uint8_t aArgsLength, char *aArgs[])
             break;
         }
     }
+    else if (StringMatch(aArgs[0], "enable"))
+    {
+        SuccessOrExit(error = Get<Radio>().Enable());
+        Output("Radio link started.\r\n");
```

Comment:
Could we remove the output string? The diag module will output `Done` if no error happens.